### PR TITLE
Deprecated usage of Object#=~ might cause pbms with Mongo

### PR DESCRIPTION
**Pull request by RobertDober**
My local tests fail, need to make the PR to see what Travis says.

That said, seems a necessary fix for Ruby 2.7.*

### CODE REVIEW

**Comment by did** [comment]
hi @RobertDober! Glad to see you here ;-)

Interesting PR, it's safer indeed to use `.to_s` all the more since I've to upgrade to a newer version of Mongo (MongoDB, Mongoid, ..etc.
Regarding your other issue (Travis secrets stuff), I've to look into it.
thanks a lot!

**Comment by RobertDober** [comment]
@did it would not be my personal preference however there might be some pragmatic reasons I would be perfectly happy with and that's why I ask: Shall I try to get rid of more warnings in this PR or shall I open a different PR for this?

Depending on your answer this PR seems ready for review.

File: .travis.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ before_script: ./script/ci/before_build.sh
 rvm:
   - 2.6.1
   - 2.6.5
+  - 2.7.1
```
Comment (did) [inline .travis.yml:7]: let's be crazy and use the latest one (2.7.2)

Comment (RobertDober) [reply .travis.yml:7]: with pleasure

Comment (RobertDober) [reply .travis.yml:7]: 3.0.0.pre1 is out though but even I am not that mad

**Pull request merged**